packages/paste-core/components/heading/stories/index.stories.tsx

```diff
@@ -1,25 +0,0 @@
-import * as React from 'react';
```

Comment:
We should keep these stories for `Heading`

Reply:
@SiTaggart I moved the stories under the Typography component, for both Heading and Paragraph. Now that I'm thinking about it though, that doesn't make sense. I'll move those back to their respective components.